packages/core/src/MapIt/index.js

```diff
@@ -419,6 +440,8 @@ MapIt.propTypes = {
   drawChildren: PropTypes.bool,
   drawGrandChildren: PropTypes.bool,
   drawProfile: PropTypes.bool,
+  marker: PropTypes.bool,
+  markerPointer: PropTypes.arrayOf(PropTypes.number),
```

Comment:
Why do we need both of these? If `markerPointer` is defined, it should mean draw the marker; otherwise don't draw it. What if I set `marker: true` but `markerPointer: undefined`?

Reply:
@kilemensi I included this because we need to specify `where` to draw the marker.  It could be wrong. I can remove marker and just work with markerPointer instead

Reply:
Yes @nerdlet lets just start with `markerPointer` alone... why would someone say _where_ to draw the marker if they don't want you to draw it?